test/nested/apps_1128_frag_native_instance_type_override.wdl

```diff
@@ -0,0 +1,113 @@
+version 1.1
+
+workflow override {
+  call create_input
+
+  call default {
+    input: files = create_input.inputs
+  }
+
+  call instance_type {
+    input: files = create_input.inputs
+  }
+
+  # Conditional, single call
+  Boolean n = true
+  if (n) {
+    call instance_type as conditional_isntance_type {
+      input: files = create_input.inputs
+    }
+  }
+
+  # scatter, single call
+  scatter (i in [1,2]) {
+    call instance_type as scatter_isntance_type {
+      input: files = create_input.inputs
+    }
+  }
+
+  # Conditional, multiple calls
+  Boolean m = true
+  if (m) {
+    call instance_type as conditional_isntance_type_1 {
+      input: files = create_input.inputs
+    }
+    call instance_type_2 as conditional_isntance_type_2 {
+      input: files = create_input.inputs
+    }
+  }
+}
+
+task create_input {
+  command <<<
+    echo "a" > input_a.txt
+    echo "b" > input_b.txt
+  >>>
+  output {
+    Array[File] inputs = ["input_a.txt", "input_b.txt"]
+  }
+}
+
+task default {
```

Comment:
It is likely some other workflow has a task with the same name, which will result in a conflict when running integration tests (the applet of a workflow compiled first will be overridden by the applet with the same name from a workflow compiled afterwards), so it's better to use more specific names.

Reply:
fixed in [15cc191](https://github.com/dnanexus/dxCompiler/pull/333/commits/15cc191bb8ec1b4c6fed9c4abe0963b3291562ce)